plugins/virtual_event/services/VirtualEventService.php

```diff
@@ -150,32 +150,25 @@ protected function validateScheduleEvents (KalturaVirtualEvent $virtualEvent)
 		$partnerId = kCurrentContext::getCurrentPartnerId();
 		if ($virtualEvent->agendaScheduleEventId)
 		{
```

Comment:
would change it to run in loop on array of virtualScheduleEvents, check if they set and call to validate instead of having 3 if conditions